[23.0] 23.0 Release notes

dannon opened the pull request:
Standing release notes PR -- please hack away at it.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

dannon committed:
Initial 23.0 release notes

dannon committed:
More date tweaks

jdavcs commented on doc/source/releases/23.0_announce_user.rst:
```diff
@@ -0,0 +1,167 @@
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
+
+<team members go here>
```
```suggestion
Alireza Heidari <https://github.com/itisAliRH>
Bryan Raubenolt <https://github.com/thepineapplepirate>
Cameron Hyde <https://github.com/neoformit>
Jennifer Hillman-Jackson <https://github.com/jennaj>
John Chilton <https://github.com/jmchilton>
John Davis <https://github.com/jdavcs>
Laila Los <https://github.com/ElectronicBlueberry>
```

dannon committed:
Update doc/source/releases/23.0_announce_user.rst

Co-authored-by: John Davis <jdavcs@gmail.com>

hexylena committed:
Add link to admin highlights

hexylena committed:
Cleanup non-relevant items and refactorings

hexylena committed:
More non-relevant refactorings

hexylena committed:
credit

hexylena committed:
Use semantic headings

hexylena committed:
Write user facing release notes

hexylena committed:
fix links, make list

hexylena committed:
Add separators, labels

Per @electronicblueberry's suggestion

dannon committed:
Note trackster deprecation

dannon committed:
Deprecation notice for myexperiment.org workflow exports

hexylena committed:
add video thanks to @delphine-l

dannon committed:
Regenerate notes to include new PRs.

dannon committed:
Cleanup in user announce post-regen

dannon committed:
Regenerate PRs

martenson committed:
update 23.1 tag and estimates

davelopez committed:
Update user facing release notes

- Update the WF Invocations Export section
- Add the History Export Tracking section

hexylena commented on doc/source/releases/23.0_announce_user.rst:
```diff
@@ -74,12 +74,17 @@ It's easier than ever to change the datatype of every dataset in a collection us
 ----------------------------
 
-Are you interested in publishing workflow artefacts? You can do that in one of two formats now, RO-Crate and BioCompute Object. These are both standards based efforts to package research data and metadata. Read more about `RO-Crate <https://www.researchobject.org/ro-crate/>`__ and `BioCompute Objects <https://biocomputeobject.org/>`__. You'll need to have a remote file source configured (e.g. FTP ...
+Galaxy now offers two options for packaging and publishing workflow artifacts: RO-Crate and BioCompute Object. These standard-based formats provide a way to bundle research data and metadata in a structured way. To learn more about these formats, check out RO-Crate <https://www.researchobject.org/ro-crate/>__ and BioCompute Objects <https://biocomputeobject.org/>__. Please note that you'll need t ...
```
I'm not sure about the added sentence, the audience isn't admins, they don't care about celery.

```suggestion
Galaxy now offers two options for packaging and publishing workflow artifacts: RO-Crate and BioCompute Object. These standard-based formats provide a way to bundle research data and metadata in a structured way. To learn more about these formats, check out `RO-Crate <https://www.researchobject.org/ro-crate/>`__ and `BioCompute Objects <https://biocomputeobject.org/>`__. You can directly export these files to any writeable remote file sources that have been configured (e.g., FTP or Dropbox).
```

the ` are additionally important for the urls, no?

hexylena commented on doc/source/releases/23.0_announce_user.rst:
```diff
@@ -74,12 +74,17 @@ It's easier than ever to change the datatype of every dataset in a collection us
+History Export Tracking
+----------------------------
+
+Good news for users with Celery enabled on their Galaxy instances: a new History export user interface is now available with additional options for generating temporal History downloads. This feature also allows you to package your history as an RO-Crate and export it to remote file sources for more permanent storage. Plus, every time you export your history to one of these remote sources, a new  ...
```
Could you add a screenshot here?

```suggestion
A new History export user interface is now available that makes it easier to download your histories. You can export your history in a variety of formats, and either download it directly, or save it to a remote file source for more permanent storage. Plus, every time you export your history to one of these remote sources, a new tracking record is created with a button that lets you re-import that history snapshot with ease.
```

again I think we can just say this without confusing users over what celery is or how they could enable it. I think we should pair this with a note in the admin facing release notes that mentions Celery is required for these new features that your users might be asking about.

davelopez replied:
You're right, I was just a bit anxious about users being confused because these features are not visible in big instances like main.

mvdbeek replied:
We'll have celery on main pretty soon, I think it's not worth stressing about this.

davelopez committed:
Apply suggestions from code review

Co-authored-by: Helena <hexylena@galaxians.org>

mvdbeek commented on doc/source/releases/23.0_announce_user.rst:
```diff
@@ -0,0 +1,181 @@
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user using the new search interface to find collection operations tools, and then a Filter tool."></iframe>
```
```suggestion
   <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user changing the datatype for a dataset collection."></iframe>
```

hexylena replied:
yeah good improvement.

hexylena committed:
Update doc/source/releases/23.0_announce_user.rst

Co-authored-by: Marius van den Beek <m.vandenbeek@gmail.com>

mvdbeek committed:
Describe Conditional Workflow Steps

mvdbeek committed:
Add 23.0 to TOC

dannon committed:
Update sample from config-rebuild

dannon committed:
Update freshly merged PRs for release notes

dannon committed:
Add configuration changes

mvdbeek commented on doc/source/releases/23.0_announce.rst:
```diff
@@ -42,6 +42,317 @@ To update an existing Galaxy repository run:
+    the location of involucro on the Galaxy host. This is ignored if the relevant
+    container resolver isn't enabled, and will install on demand unless
+    involucro_auto_init is set to false.
+
```
```suggestion
```

mvdbeek commented on doc/source/releases/23.0_announce.rst:
```diff
@@ -42,6 +42,317 @@ To update an existing Galaxy repository run:
+    notifies itself of new jobs via in-memory queues.  Jobs are run locally on
+    the system on which Galaxy is started.  Advanced job running capabilities can
+    be configured through the job configuration file or the <job_config> option.
+
```
```suggestion
```

mvdbeek commented on doc/source/releases/23.0_announce.rst:
```diff
@@ -42,6 +42,317 @@ To update an existing Galaxy repository run:
+    through the Admin control panel -- see "Manage Allowlist"
+
+    The value of this option will be resolved with respect to <managed_config_dir>.
+
```
```suggestion
```

mvdbeek commented on doc/source/releases/23.0_announce.rst:
```diff
@@ -42,6 +42,317 @@ To update an existing Galaxy repository run:
+    If this option is set to none or an invalid path, installing tools with dependencies
+    from the Tool Shed or in Conda will fail.
+
+
```
```suggestion
```

mvdbeek reviewed (commented):
Don't think we should list the description changes.

dannon committed:
Apply suggestions from code review

Co-authored-by: Marius van den Beek <m.vandenbeek@gmail.com>

dannon committed:
Stable PRs regen

dannon committed:
Add administrative notes

dannon committed:
Highlights blurb

dannon committed:
Fix typo
